google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/CopyJobConfiguration.java

```diff
@@ -19,6 +19,7 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 import com.google.api.services.bigquery.model.JobConfigurationTableCopy;
+import com.google.cloud.bigquery.LoadJobConfiguration.Builder;
```

Comment:
How is this used? I find it a little strange that a copy configuration uses load config builder.

Reply:
not used, removed.